src/main/java/io/inscopemetrics/kairosdb/aggregators/HistogramFilterAggregator.java

```diff
@@ -124,45 +125,21 @@ public DataPointGroup aggregate(final DataPointGroup dataPointGroup) {
 
     @Override
     public boolean canAggregate(final String groupType) {
-        return HistogramDataPointFactory.GROUP_TYPE.equals(groupType);
+        return HistogramDataPoint.GROUP_TYPE.equals(groupType);
     }
 
     @Override
     public String getAggregatedGroupType(final String groupType) {
-        return HistogramDataPointFactory.GROUP_TYPE;
-    }
-
-    static double truncate(final double val) {
-        final long mask = 0xffffe00000000000L;
-        return Double.longBitsToDouble(Double.doubleToRawLongBits(val) & mask);
-    }
-
-    /**
-     * Gives an inclusive bound of the bin that the passed in value will be placed in.
-     * If the value is positive, it will be an inclusive upper bound, and if it is negative,
-     * it will be an inclusive lower bound.
-     *
-     * @param val the value whose bucket bound will be calculated
-     * @return the inclusive upper or lower bound of the bin
-     */
-    static double binInclusiveBound(final double val) {
-        long bound = Double.doubleToLongBits(val);
-        // TODO(ville): Extract the magic numbers and if necessary add documentation.
-        // CHECKSTYLE.OFF: MagicNumber -
-        bound >>= 45;
-        bound += 1;
-        bound <<= 45;
-        bound -= 1;
-        // CHECKSTYLE.OFF: MagicNumber
-
-        return Double.longBitsToDouble(bound);
+        return HistogramDataPoint.GROUP_TYPE;
     }
 
     private static boolean isNegative(final double value) {
+        // TODO(ville): Add documentation to why this is implemented this way.
```

Comment:
Do you intend to resolve before merging?

Reply:
Nope. Frankly, on the surface I don't know why it was written this way. Joey will be here in a few weeks. Hopefully he remembers ;)